app/assets/javascripts/admin/modules/document-history.js

```diff
@@ -0,0 +1,38 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function DocumentHistory (module) {
+    this.module = module
+  }
+
+  DocumentHistory.prototype.init = function () {
+    this.setupEventListeners()
+  }
+
+  DocumentHistory.prototype.setupEventListeners = function () {
+    var documentHistoryTab = this.module.closest('.govuk-tabs').querySelector('#history_tab')
+
+    documentHistoryTab.addEventListener('click', function (e) {
+      var newerLink = documentHistoryTab.querySelector('.app-view-document-history-tab__newer-pagination-link')
+      var olderLink = documentHistoryTab.querySelector('.app-view-document-history-tab__older-pagination-link')
+
+      if (newerLink && newerLink.contains(event.target)) {
+        e.preventDefault()
+
+        window.fetch(new URL(document.location.origin + newerLink.dataset.remotePagination))
```

Comment:
Do we have a polyfill for fetch? 

Do we know what happens when this promise fails?

Reply:
@kevindew with regards to IE11 support, [the GDS service manual says](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in-from-june-2022):

> You will still need to support Internet Explorer 11 if your analytics data shows at least 2% of your users arriving at the service’s start page are using it.

According to Analytics data for Whitehall, IE11 usage [dropped off a cliff](https://analytics.google.com/analytics/web/#/report/visitors-browser/a26179049w83104518p86186851/_u.date00=20220101&_u.date01=20221231&explorer-table.plotKeys=%5B%5D&_r.drilldown=analytics.browser:Internet%20Explorer&_.gSectionId=summary&explorer-graphOptions.selected=analytics.nthMonth) throughout the course of last year:

<img width="687" alt="Number of IE11 users" src="https://user-images.githubusercontent.com/7735945/211377091-d3621b7e-a369-4e8e-bea4-21ece238ef57.png">

_(admittedly it was a small cliff to begin with)_

As of December 2022, **0.26%** of Whitehall users were on IE – so well below the 2% threshold required of public facing services.

It's hard to know when to draw a line, but I'm wondering whether it's now safe to assume that the user's browser supports `window.fetch` without requiring a polyfill? It's been [supported by](https://caniuse.com/fetch) evergreen browsers for the past 8 years now.

Reply:
@ollietreend I think it's fine for us to drop IE11 support in a considered way but I don't think we should just do it casually without some involvement from wider GOV.UK frontenders (which I believe @dnkrj is exploring at the moment). 

The complicating factors seem to be:

a) deciding what not supporting IE means, there's a decision as to whether it's a "we don't care if it's broken" or if it's a "we purposefully degrade it to fall back to plain HTML" so it can still kinda work with progressive enhancement (there's more on this in [design system issue 2607](https://github.com/alphagov/govuk-frontend/discussions/2607) which talks about functional/compliant browser definitions)
b) we're still part of a GOV.UK that does support IE11 - naturally the direction of travel is to end support, so it's ok to lead that way, but shouldn't just diverge without coordination and communication.

Reply:
Thinking pragmatically, if we’re happy this is a progressive enhancement, then a short term approach could be to use object detection as a way to use fetch without opening up errors for IE

I.e. we could have the start of the module exit early if window.fetch is falsey. It’d then just fall back to no-js behaviour.

Reply:
As discussed when the promise fails it falls back to the non-js behaviour and reloads the page with the next or previous set of history.